unleash/Unleash.swift

```diff
@@ -81,6 +81,19 @@ public class Unleash {
     }
     
     public func isEnabled(name: String) -> Bool {
-        return toggles?.features.first { $0.name == name }?.enabled ?? false
+        var result = false
+        
+        let toggle_strategies = toggles?.features.first { $0.name == name }?.strategies
+        
+        for strategy in toggle_strategies! {
+            if result == true { break; }
+            
+            let targetStrategy = strategies.first { $0.name == strategy.name }
+            let targetResult = targetStrategy?.isEnabled(parameters: strategy.parameters!)
+            
+            result = result || targetResult!
```

Comment:
Oops forgot I had done this. This was before the `break`. It was running every item in the loop and only updating result if it became true.

Reply:
so why not get rid of the result var and just return from inside the loop if `targetResult == true`?

Reply:
I did try that. I can `return true` when I break `toggle_strategies` into its own line separate from the `for` loop. When I try to do:
```
for strategy in toggles?.features.first { $0.name == name }?.strategies! {
  ...
  return true
}
```
I get an error about "Anonymous closure argument not contained in a closure." I get that error a lot, actually.

Reply:
what about if you provide a `return false` after the loop. That doesn't seem to address the compile error, but I could see the compiler complaining if one path did not return a value

Reply:
You'll see the "Anonymous closure argument not contained in a closure" error when you are accessing parameters by number outside the scope of a closure.

Ex.
```
func youCantSeeMe(param1: String, param2: String) {
  // insert some logic here

  // here, the compiler will give you that warning because our numbered parameters are not inside the scope of a
  // closure
  print($0, $1, param1, param2)
}
```

Ex 2.
```
func featuresActivated(features: [Features]) -> [Features] {
  // Here, the numbered argument is used inside the scope of the filter closure.
  return features.filter { $0.isActive }
}
```